Stitch/Home/View/ProjectsHomeView.swift

```diff
@@ -32,6 +46,11 @@ struct ProjectsHomeView: View {
 
     var body: some View {
         VStack {
+            // Search Bar
+            TextField("Search Projects...", text: $searchQuery)
+                .padding()
+                .textFieldStyle(RoundedBorderTextFieldStyle())
+                .padding(.horizontal)
```

Comment:
Let's surround this with `DEV_DEBUG` until we get an idea for how we want to polish this. Otherwise I'm a big fan of having this!!

```suggestion
#if DEV_DEBUG
            TextField("Search Projects...", text: $searchQuery)
                .padding()
                .textFieldStyle(RoundedBorderTextFieldStyle())
                .padding(.horizontal)
#endif
```